moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -524,6 +526,19 @@ class MoveGroupInterface::MoveGroupInterfaceImpl
     support_surface_ = support_surface;
   }
 
+  void setPlaceEEF(const bool place_eef)
+  {
+    place_eef_ = place_eef;
+    ROS_INFO_NAMED("move_group_interface", "Place EEF: %s", place_eef_ ? "yes" : "no");
```

Comment:
Not many of the other functions have logs like this. I'm fine with it, but it should be changed to debug, it'll probably just add to the noise most of the time.